src/yb/master/backfill_index.cc

```diff
@@ -645,20 +645,8 @@ BackfillTable::BackfillTable(
       indexed_table_(indexed_table),
       index_infos_(indexes),
       requested_index_ids_(IndexIdsFromInfos(indexes)),
+      requested_index_names_(master->catalog_manager()->RetrieveIndexNames(requested_index_ids_)),
```

Comment:
I'm not too sure if I'd want this function in catalog_manager.cc

Seems like this is something we'd only use for backfill_indexes and may be polluting catalog manager (which is already doing too many things).

have you considered,  adding it in this file in an anonymous namespace, as 

```
namespace {

std::string RetrieveIndexNames(CatalogManager *mgr, const std::unordered_set<std::string>& requested_index_ids) {
...
}

};

Reply:
@amitanandaiyer Sure, sounds fair. I've made the change accordingly. It looks neater now, actually.